docs/guides/cloud/projects.mdx

```diff
@@ -197,161 +205,42 @@ You can create multiple Record Keys for a project, or delete existing ones from
 [Cypress Cloud](https://on.cypress.io/cloud).
 
 <DocsImage
-  src="/img/dashboard/record-keys-in-project-settings-dashboard.png"
+  src="/img/guides/cloud/projects/record-keys-in-project-settings-dashboard.jpg"
   alt="Record key in project settings"
 />
 
 You can also find your Record Key inside of the _Settings_ tab in the Cypress
 App.
 
 <DocsImage
-  src="/img/dashboard/record-key-shown-in-desktop-gui-configuration.jpg"
+  src="/img/guides/cloud/projects/record-key-desktop-gui.jpg"
   alt="Record Key in Configuration Tab"
 />
 
-## Record keys
```

Comment:
did you check onlinks to make sure any of the headers that were changed/removed dont have shortcuts to them?

Reply:
Good call. It looks there are a lot of old dashboard/cloud "on" links that could be updated. I'll add a separate task for that after this update.